src/main.ts

```diff
@@ -1,32 +1,47 @@
 import * as blessed from 'blessed'
 import * as contrib from 'blessed-contrib'
-import { screenOption, scrollOption, navigateOption, listStyle } from './options'
+import {
+  screenOption,
+  scrollOption,
+  navigateOption,
+  listStyle,
+}                   from './options'
 
 const screen = blessed.screen(screenOption)
 screen.title = 'wechaty-cli'
 
-const grid = new contrib.grid({rows: 12, cols: 12, screen: screen})
+const grid = new contrib.grid({
```

Comment:
I don't know how to satisfy the requirement, since node-facenet and the example code of blessed-contrib also use lowercase constructor name.